provisioner/k8s/README.md

```diff
@@ -1,3 +1,112 @@
-# k8s
+# k8s Provisioner
 
-A concrete Provisioner implementation for "plain" k8s bundles.
+## Summary
+
+The k8s provisioner is a core rukpak provisioner that knows how to interact with `k8s` format bundles. 
+The `k8s` bundles, or plain bundles, are simply container images containing a set of static kubernetes YAML manifests in a given directory.  
+For example, below is an example Dockerfile that builds a plain `k8s` bundle from a directory containing static manifests.  
+```dockerfile
+FROM scratch
+LABEL manifests="/manifests"
+COPY ./manifests /manifests
+```
+
+The `k8s` provisioner is able to unpack a given `k8s` bundle onto a cluster and then instantiate it, making the
+content of the bundle available in the cluster. It does so by reconciling the `Bundle` and `BundleInstance` types that 
+have the `spec.provisionerClassName` field set to `core.rukpak.io/plain`. This field must be set to the correct provisioner name 
```

Comment:
+1 to the idea of renaming this to the `plain` provisioner since that is its className.

Reply:
tracking this in #87